### PR TITLE
Add a basic README file

### PR DESCRIPTION
**Pull request by mborgerson**
This PR adds a basic README describing what cromwell is and does. We can improve this as we go forward but I think it's a good base.

The text of this README originates from the Cromwell Manual, written by
David Pye. The original Cromwell Manual can be found at [this
archived Xbox Linux Project wiki page](https://web.archive.org/web/20050812082916/http://www.xbox-linux.org/wiki/Cromwell_Manual).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cromwell
+========
+Cromwell is a free, legal BIOS replacement for the Xbox. It is capable of
```
Comment (JayFoxRox) [inline README.md]: Same comments as for #3: "original Xbox" is more suitable by now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* Flashed to a modchip: If selected, it will let you boot linux. If not
+  selected, it will load the original MS BIOS, so no functionality is lost.
+* Flashed to the TSOP: this replaces the MS BIOS stored on the Xbox, and makes
+  it impossible to play games (whether original or not).
+
+The advantage of the second method is that it makes it possible to dedicate the
+whole harddisk to Xbox Linux.
+
+Xromwell is a very similar program, built from the same sourcecode. However,
+instead of being a BIOS, it is an XBE (XBox Executable), meaning that it can be
```
Comment (JayFoxRox) [inline README.md]: XBox &rarr; Xbox

File: README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+It can be installed and used in one of two ways:
+
+* Flashed to a modchip: If selected, it will let you boot linux. If not
+  selected, it will load the original MS BIOS, so no functionality is lost.
+* Flashed to the TSOP: this replaces the MS BIOS stored on the Xbox, and makes
+  it impossible to play games (whether original or not).
+
+The advantage of the second method is that it makes it possible to dedicate the
+whole harddisk to Xbox Linux.
```
Comment (JayFoxRox) [inline README.md]: Xbox Linux &rarr; Xbox-Linux (?)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+cromwell
+========
+Cromwell is a free, legal BIOS replacement for the Xbox. It is capable of
+loading Linux from the Xbox's Hard Disk, and CDROM. It is not able to load Xbox
```
Comment (JayFoxRox) [inline README.md]: CDROM &rarr; CD-ROM

File: README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+cromwell
+========
```
Comment (JayFoxRox) [inline README.md]: Blank line after this would help to mark this as headline when not using markdown viewer.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+Cromwell is a free, legal BIOS replacement for the original Xbox. It is capable
+of loading Linux from the Xbox's Hard Disk, and CD-ROM. It is not able to load
+original Xbox games, neither originals nor copies.
+
+It can be installed and used in one of two ways:
+
+* Flashed to a modchip: If selected, it will let you boot linux. If not
```
Comment (binarymaster) [inline README.md]:
```suggestion
* Flashed to a modchip: If selected, it will let you boot Linux. If not
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  selected, it will load the original MS BIOS, so no functionality is lost.
+* Flashed to the TSOP: this replaces the MS BIOS stored on the Xbox, and makes
+  it impossible to play games (whether original or not).
+
+The advantage of the second method is that it makes it possible to dedicate the
+whole hard disk to Xbox-Linux.
+
+Xromwell is a very similar program, built from the same source code. However,
```
Comment (binarymaster) [inline README.md]:
```suggestion
**Xromwell** is a very similar program, built from the same source code. However,
```

**Comment by thrimbor** [comment]
I think this is good enough to merge as-is - we can always refine it later (or add additional info to the wiki). Merging, thanks!

**Pull request merged**